docs/deployment/file-access-auth.md

```diff
@@ -0,0 +1,159 @@
+---
+title: File Access Authorization
+parent: Deployment
+nav_order: 4
+---
+
+# (BETA) File Access Authorization
+
+> **IMPORTANT:** This feature is currently in beta. Configuration and log formats are subject to change.
+
+> **IMPORTANT:** This feature is only supported on macOS 13 and above.
+
+File Access Authorization allows admins to configure Santa to monitor filesystem paths for potentially unwanted access and optionally deny the operation.
+
+## Enabling the Feature
+
+To enable this feature, the `FileAccessPolicyPlist` key in the main Santa configuration must contain the path to a configuration file (format specified below).
```

Comment:
When you say "format specified below" here, do you mean the ## Configuration section or the ## Example Configuration? Might be worth linking to either, e.g.:

"... a configuration file. See the format specified in the [Example Configuration](#example-configuration)."